testling + centos + phantomjs getting error: No headless browser found

I'm having a difficult time getting my setup to work, although everything seems to be in order.  I have PhantomJS installed:

```
-bash-4.1$ node node_modules/testling/node_modules/browser-launcher/example/detect.js
# available browsers:
[ { command: 'phantomjs',
    version: '1.9.7',
    name: 'phantom',
    re: /(\S+)/,
    type: 'phantom',
    headless: true,
    profile: true },
  { command: 'firefox',
    version: '24.3.0',
    name: 'firefox',
    re: /Mozilla Firefox (\S+)/,
    type: 'firefox',
    profile: true } ]
```

It works:

```
-bash-4.1$ phantomjs tests/index.js
Error: Cannot find module 'assert'

  phantomjs://bootstrap.js:289
  phantomjs://bootstrap.js:254 in require
  tests/index.js:13
```

but yet testling can't find it:

```
-bash-4.1$ make coverage
./node_modules/.bin/browserify -t coverify tests/index.js | testling | ./node_modules/.bin/coverify
No headless browser found.
```

Any suggestions?

--- a/readme.markdown
+++ b/readme.markdown
@@ -116,6 +116,10 @@ ok 1 should be equal
 
 The exit code of coverify is non-zero when there are unreachable expressions.
 
+# dependencies
+
+Make sure you have [PhantomJS](https://github.com/ariya/phantomjs) installed; this is the headless browser that testling will run your tests in if you are not using the `-u` option.
+
 # install
 
 First, install `browserify` globally so that the `testling` command can find it

